clients/runtime/src/lib.rs

```diff
@@ -111,13 +139,12 @@ pub struct WrapperKeepOpaque<T> {
 pub struct SpacewalkRuntime;
 
 impl Config for SpacewalkRuntime {
-	type Index = Index;
-	type BlockNumber = BlockNumber;
+	type AssetId = ();
```

Comment:
What is the `AssetId` used for? Is it okay to leave it empty?

Reply:
It is a new field from the `subxt` `Config` trait, but as the comment of this type [says](https://github.com/paritytech/subxt/blob/c3267ed488b0fcee5f537c60c341cc5f1c7d63e0/core/src/config/mod.rs#L61), I believe it is not relevant for our standalone testchain.